app/controllers/reports_controller.rb

```diff
@@ -185,6 +198,20 @@ def checks
     Aces::MecCheck.where(created_at: range).or(updated_at: range)
   end
 
+  def pdm_checks
+    transactions = if @app.present?
```

Comment:
Optional: I would rename `@app` to `@application_id` if possible.

Reply:
That should be fine we are following the pattern from before.